pom.xml

```diff
@@ -195,6 +200,14 @@
         <plugin>
           <artifactId>maven-surefire-plugin</artifactId>
           <version>2.18.1</version>
+          <dependencies>
+            <dependency>
+              <!-- Force junit provider because surefire will choose the testng one if testng is detected -->
+              <groupId>org.apache.maven.surefire</groupId>
+              <artifactId>surefire-junit47</artifactId>
```

Comment:
Forget it for the moment: we need to see how TestNG will fix its issue with JUnit first.
I made some tests with junit/testng abstract classes and junit=true/false, and everything is working as expected (unlike we decide to change the behavior) with testng classes. I think TestNG must be consistent between testng and junit classes before changing something in surefire :)